internal/indexer/mocks.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stellar/wallet-backend/internal/indexer/types"
 )
 
-type EffectsProcessorInterface interface {
+type OperationProcessorInterface interface {
 	ProcessOperation(ctx context.Context, opWrapper *operation_processor.TransactionOperationWrapper) ([]types.StateChange, error)
```

Comment:
nit: I actually forgot to move this to indexer.go file but normally interfaces should be defined in the place they are used. Can you please move this there with the other interfaces